packages/cli/src/commands/account/set-wallet.ts

```diff
@@ -0,0 +1,57 @@
+import { flags } from '@oclif/command'
+import { BaseCommand } from '../../base'
+import { newCheckBuilder } from '../../utils/checks'
+import { displaySendTx } from '../../utils/cli'
+import { Flags } from '../../utils/command'
+
+export default class SetWallet extends BaseCommand {
+  static description =
+    "Sets the wallet of a registered account on-chain. An account's wallet is an optional wallet associated with an account. Can be set by the account or an account's signer."
+
+  static flags = {
+    ...BaseCommand.flags,
+    account: Flags.address({ required: true }),
+    wallet: flags.string({ required: true }),
```

Comment:
This should be an address too, right?
```suggestion
    wallet: Flags.address({ required: true }),
```

Reply:
yes, thanks!